backend/src/test/java/gov/cdc/usds/simplereport/service/DataHubUploaderServiceTest.java

```diff
@@ -83,23 +102,25 @@ void serialization() {
             + "\",\"986543321\",\"000111222\",\"UNK\",\""
             + date
             + "\",\"260415000\",\"F\",\"123456\",\"Washington\",\"Washington\",\"Imaginary Site\",\"555-867-5309\",\"DC\",\"736 Jackson PI NW\",\"\",\"20503\"\n"
-            + "\"\",\""
+            + "\""
+            + te2.getPriorCorrectedTestEventId()
+            + "\",\""
             + date
-            + "\",\"SFN\",\"UNK\",\"UNK\",\"\",\""
+            + "\",\"SFN\",\"Y\",\"UNK\",\"\",\""
             + instrumentId2
             + "\",\""
             + date
-            + "\",\"54321-BOOM\",\"Washington\",\"Washington\",\"facility@test.com\",\"Imaginary Site\",\"555-867-5309\",\"DC\",\"736 Jackson PI NW\",\"\",\"20503\",\"DOOOOOOM\",\"Washington\",\"Washington\",\"Doctor\",\"Doom\",\"800-555-1212\",\"DC\",\"736 Jackson PI NW\",\"\",\"20503\",\"The Mall\",\"\",\""
+            + "\",\"54321-BOOM\",\"Washington\",\"Washington\",\"facility@test.com\",\"Imaginary Site\",\"555-867-5309\",\"DC\",\"736 Jackson PI NW\",\"\",\"20503\",\"DOOOOOOM\",\"Washington\",\"Washington\",\"Doctor\",\"Doom\",\"800-555-1212\",\"DC\",\"736 Jackson PI NW\",\"\",\"20503\",\"The Mall\",\"18990510\",\""
             + patientId2
-            + "\",\"\",\"\",\"\",\"U\",\"John\",\"U\",\"Boddie\",\"Brown\",\"503-867-5309\",\"UNK\",\"STAFF\",\"\",\"\",\"\",\"Jr.\",\"\",\"P\",\"UNK\",\""
+            + "\",\"Washington\",\"Washington\",\"\",\"N\",\"Fred\",\"M\",\"Astaire\",\"\",\"5168675309\",\"Spanish\",\"2106-3\",\"RESIDENT\",\"DC\",\"736 Jackson PI NW\",\"\",\"\",\"123\",\"20503\",\"P\",\"N\",\"" // "\",\"\",\"\",\"\",\"U\",\"John\",\"U\",\"Boddie\",\"Brown\",\"503-867-5309\",\"UNK\",\"STAFF\",\"\",\"\",\"\",\"Jr.\",\"\",\"P\",\"UNK\",\""
```

Comment:
Stray comment?

Reply:
Ah yeah, gross - this is from when I was adjusting it

Reply:
Fixed in 7aa82a3